packages/bruno-cli/docker/smoke-test.sh

```
@@ -0,0 +1,131 @@
+  ENV_FLAG=""
+  if [ -n "$COLLECTION_ENV" ]; then
+    ENV_FLAG="--env $COLLECTION_ENV"
+  fi
+  echo "Test 5: bru run $RUN_TARGET${COLLECTION_ENV:+ --env $COLLECTION_ENV}"
+  echo "----- bru run output -----"
+
+  set +e
+  OUTPUT=$(docker run --rm \
+    -v "$COLLECTION_PATH:/bruno" \
+    "$IMAGE" \
+    run "$RUN_TARGET" $ENV_FLAG 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)